lib/symbols.js

```diff
@@ -30,6 +30,7 @@ const keys = {
   kRequestValidateFns: Symbol('fastify.request.cache.validateFns'),
   kRequestPayloadStream: Symbol('fastify.RequestPayloadStream'),
   kRequestAcceptVersion: Symbol('fastify.RequestAcceptVersion'),
+  kRequestValidateWeakMap: Symbol('fastify.request.cache.validators'),
```

Comment:
Instead of adding, was not expected to use `kRequestValidateFns` instead?

Reply:
Hmm, yeah you're right. I reacted too fast that I missed the other one.
I'll open a PR to clean up this, thanks for taking a second look 🙇 

Thankfully the issue is solved, now just a matter of cleanup the mess